src/migrations/20221018174734-remove-objective-roles.js

```diff
@@ -0,0 +1,105 @@
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.transaction(
+      async (transaction) => {
+        await queryInterface.dropTable('ActivityReportObjectiveRoles', { transaction });
+        await queryInterface.dropTable('ObjectiveRoles', { transaction });
+      },
```

Comment:
We also need to clean up the audit log artifacts. Otherwise, the rollback will produce errors, e.g.
`ERROR: function "ZALNoUpdateFObjectiveRoles" already exists with same argument types`

Reply:
like [this](https://github.com/HHS/Head-Start-TTADP/pull/1092/commits/9c19ee411c9554ab8c19ed65c5b8fee7134564f6)?

Reply:
yikes obviously not, ok